ts/App.tsx

```diff
@@ -9,9 +9,37 @@ import { GestureHandlerRootView } from "react-native-gesture-handler";
 import { SafeAreaProvider } from "react-native-safe-area-context";
 import { Provider } from "react-redux";
 import { PersistGate } from "redux-persist/integration/react";
+import * as Sentry from "@sentry/react-native";
+import { ErrorEvent, TransactionEvent } from "@sentry/types";
 import RootContainer from "./RootContainer";
 import { persistor, store } from "./boot/configureStoreAndPersistor";
 import { LightModalProvider } from "./components/ui/LightModal";
+import { sentryDsn } from "./config";
+import { isLocalEnv } from "./utils/environment";
+
+const removeUserFromEvent = (event: ErrorEvent | TransactionEvent) => {
+  // console.log(JSON.stringify(event));
+  // Modify or drop the event here
+  if (event.user) {
+    // Don't send user's email address
+    return { ...event, user: undefined };
+  }
+  return event;
+};
+
+Sentry.setUser(null);
```

Comment:
Isn't better to move this to index.js?

Reply:
Following the setup guide/automated CLI, I opted to put all the initial configuration of sentry directly on the root App definition file such as the init callback